llvm/lib/Target/AIE/AIE2TargetMachine.cpp

```diff
@@ -82,6 +96,19 @@ AIE2TargetMachine::AIE2TargetMachine(const Target &T, const Triple &TT,
   setGlobalISelAbort(GlobalISelAbortMode::Enable);
 }
 
+/// Predicate for Internalize pass.
```

Comment:
Nit: I feel the comment does not help much. Instead, maybe give a rationale about what should be preserved and why?